packages/api/blocks-api/src/blocks/decorators/block-index-target.ts

```diff
@@ -0,0 +1,6 @@
+export function BlockIndexTarget(): ClassDecorator {
```

Comment:
Is this decorator necessary? If yes, we need a better name, maybe `DependencyTargetEntity`

Reply:
yes, the naming is important:
Currently, only blocks can have dependencies, but the next step (after this PR) is to add dependencies to any entity. `DependencyTargetEntity` sounds good. Not Requiring the decorator better :)

Reply:
No, it's not necessary anymore since the TargetIdentifier is gone. So I removed it